app/views/sessions/new.html.haml

```diff
@@ -1,15 +1,16 @@
 %section.sign_in.container
   .row
-    .col-sm-10.col-sm-offset-1
-      = bootstrap_form_tag layout: :horizontal,
-                           url: log_in_path,
-                           class: 'sign_in',
-                           label_col: 'col-sm-2',
-                           control_col: 'col-sm-10' do |f|
-        %header
-          %h1 Sign in
-        = f.email_field :email, label: "Email Address"
-        = f.password_field :password, label: "Password"
-        = f.form_group do
-          = f.submit "Log In"
-          = link_to "Forgot Password?", forgot_password_path, id: 'forgot_password'
+    = form_tag log_in_path, class: 'log_in' do
+      %header
+        %h1 Log in
+      .form-group.row
+        = label_tag :email, "Email Address", class: 'col-form-label col-sm-2'
+        .col-sm-6
+          = email_field_tag :email, @email, class: 'form-control'
+      .form-group.row
+        = label_tag :password, "Password", class: 'col-form-label col-sm-2'
+        .col-sm-6
+          = password_field_tag :password, nil, class: 'form-control'
+      .form_group.row
```

Comment:
`.row` without column(s) as a parent creates a negative margin of 15px on the left and right I believe. If you notice oddly out of line elements, that is why.